cmake/reports.cmake

```diff
@@ -4,13 +4,21 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
+set(pm_defined "$<TARGET_EXISTS:partition_manager>")
+set(pm_config
+  "$<${pm_defined}:$<TARGET_PROPERTY:partition_manager,PM_CONFIG_FILES>>")
+set(pm_depends
+  "$<${pm_defined}:$<TARGET_PROPERTY:partition_manager,PM_DEPENDS>>")
+
 add_custom_target(
   partition_manager_report
   COMMAND
   ${PYTHON_EXECUTABLE}
   ${ZEPHYR_BASE}/../nrf/scripts/partition_manager_report.py
-  --input ${CMAKE_BINARY_DIR}/partitions.yml
+  --input ${pm_config}
```

Comment:
This is a hack which was here already. We have the use the `--quiet` argument so that the script will not cause any error output when the `partition_manager` target does not exist.

Reply:
```
    if args.quiet:
        sys.exit(0)
```

Reply:
accepted as we now has a followup task on this.